src/store/tools/useAnnotationTool.ts

```diff
@@ -127,26 +142,45 @@ export const useAnnotationTool = <
   const activateTool = () => true;
   const deactivateTool = () => {};
 
-  const serialize = () =>
-    toolIDs.value
+  const serialize = () => {
+    const toolsSerialized = toolIDs.value
       .map((toolID) => toolByID.value[toolID])
       .filter((tool) => !tool.placing)
       .map(({ imageID, ...rest }) => ({
-        imageID: getDataID(imageID), // If parent image is DICOM, save VolumeKey
+        // If parent image is DICOM, save VolumeKey
+        imageID: getDataID(imageID),
         ...rest,
       }));
 
+    return {
+      tools: toolsSerialized,
+      labels: labels.labels.value,
+    };
+  };
+
+  type Serialized = {
+    tools: PartialWithRequired<Tool, 'imageID'>[];
+    labels: Labels<Tool>;
+  };
   function deserialize(
     this: Store,
-    serialized: PartialWithRequired<Tool, 'imageID'>[],
+    serialized: Serialized | undefined,
```

Comment:
can use `Maybe<Serialized>` here, since we don't differentiate between null and undefined.